helmfile/values/opendistro-es.yaml.gotmpl

```diff
@@ -39,17 +57,23 @@ elasticsearch:
     name: internal-elasticsearch-plugins-local
 
   extraInitContainers:
-  - name: install-s3
+  - name: install-object-storage-plugin
     image: amazon/opendistro-for-elasticsearch:1.10.1
     command:
     - sh
     - -c
     - |
+      {{ if eq .Values.objectStorage.type "s3" -}}
       /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch repository-s3
```

Comment:
Should we add `.. || exit 1` here as well?

Reply:
Yes, nice catch! I will sneak that in when I squash the commits